app/src/main/java/com/ucmobiledevelopment/freeways/ReportIncidentActivity.kt

```diff
@@ -18,21 +17,22 @@ class ReportIncidentActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
-            FreeWaysTheme() {
-                // A surface container using the 'background' color from the theme
+            FreeWaysTheme {
+                //A surface container using the 'background' color from the theme
```

Comment:
Parentheses are not required as I ran it through the IDE debugger and IDE suggestion. The spacing for the comment is according to the Kotlin convention. https://kotlinlang.org/docs/coding-conventions.html#horizontal-whitespace